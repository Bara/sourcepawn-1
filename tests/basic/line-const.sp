#include <shell>

public main() {
    int iLine = __LINE__;

    printnum(
        __LINE__
    );
    printnum(iLine);
    printnum(__LINE__);
}