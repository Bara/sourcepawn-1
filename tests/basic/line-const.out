10
7
13
13
